.github/CODEOWNERS

```diff
@@ -10,6 +10,9 @@ DEPS @brave/deps-reviewers
 script/deps.py @brave/deps-reviewers
 package*.json @brave/js-deps-reviewers
 
+# sources.gni
+**/sources.gni @brave/sources-gni-reviewers
```

Comment:
There are gni files that don't conform to this naming convention ex. `//brave/browser/tab_ui/android/brave_tab_ui_sources.gni` and there are files that add deps only ex.`//browser/extensions/resources.gni` or also only added visibility ex `//brave/third_party/tflite/tflite.gni`
We should consider general gni reviewer by `*.gni`?

Reply:
good idea